mobile/src/containers/GameSessionContainer.js

```diff
@@ -55,12 +57,12 @@ const GameSessionContainer = ({ children }) => {
     const [teamMember, setTeamMember] = useState(null)
 
     useEffect(() => {
-        loadLocalGameSession().then((localGameSession) => {
-            if (localGameSession) {
-                setGameSession(localGameSession.gameSession)
-                setTeamId(localGameSession.teamId)
-                setTeamMember(localGameSession.teamMember)
-                setGameCode(localGameSession.gameCode)
+        loadLocalGameSession().then((localStorage) => {
+            if (localStorage) {
+                setGameSession(localStorage.gameSession)
+                setTeamId(localStorage.teamId)
+                setTeamMember(localStorage.teamMember)
+                setGameCode(localStorage.gameSession.gameCode)
```

Comment:
changing this allows the initial load to work more often. the team id and team member still get deleted by nulls often